tool/tsh/tsh.go

```diff
@@ -501,6 +504,13 @@ func Run(args []string, opts ...cliOption) error {
 
 	config := app.Command("config", "Print OpenSSH configuration details")
 
+	// config-proxy is a wrapper to ensure Windows clients can properly use
+	// `tsh config`. As it's not intended to run by users directly and may
+	// not have a stable CLI interface, hide it.
+	configProxy := app.Command("config-proxy", "ProxyCommand wrapper for SSH config as generated by `config`").Hidden()
+	configProxy.Arg("target", "Target node name").Required().StringVar(&cf.ConfigProxyTarget)
+	configProxy.Arg("cluster-name", "Cluster bane").Required().StringVar(&cf.SiteName)
```

Comment:
```suggestion
	configProxy.Arg("cluster-name", "Target cluster name").Required().StringVar(&cf.SiteName)
```

Reply:
Well, that's just an embarrassing typo on my part. Thanks!